src/plugin/regions/region.js

```diff
@@ -606,13 +606,33 @@ export class Region {
         });
     }
 
+    /**
+     * @example
+     * onResize(-5, 'start') // Moves the start point 5 seconds back
+     * onResize(0.5, 'end') // Moves the end point 0.5 seconds forward
+     *
+     * @param {number} delta How much to add or subtract, given in seconds
+     * @param {string} direction 'start 'or 'end'
+     */
     onResize(delta, direction) {
         if (direction === 'start') {
+            // Check if changing the start by the given delta would result in the region being smaller than minLength
+            // Ignore cases where we are making the region wider rather than shrinking it
+            if (delta > 0 && Math.abs(this.end - (this.start + delta) < this.minLength)) {
```

Comment:
sorry, what's the call to `Math.abs` doing here?

Reply:
Wanted to make sure there's no case where start + delta is smaller than end, but maybe there's no point in checking. Also I see now there's a typo on this line. I'll just remove both Math.abs checks.